chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,72 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate menulist with locales
+	 */
```

Comment:
I'm confused. So do we want to move the preferred/fallback locales to the top of the list for easy selection or no?